slik/preprocessing.py

```diff
@@ -83,14 +84,26 @@ def change_case(dataframe=None ,column=None,case='lower'):
         raise TypeError(errstr)
 
     df = dataframe.copy()
+
+    def capitalize_case(x):
```

Comment:
Trying to understand why you wrote a function for this. `str.capitalize()` should work